umap/views.py

```diff
@@ -1012,17 +1012,16 @@ def render_to_response(self, context, **response_kwargs):
             with open(path, "rb") as f:
                 # Should not be used in production!
                 response = HttpResponse(f.read(), content_type="application/geo+json")
-            response["Last-Modified"] = self.last_modified
+            response["X-Umap-Version"] = self.version
             response["Content-Length"] = statobj.st_size
         return response
 
 
 class DataLayerVersion(DataLayerView):
     @property
     def path(self):
-        return "{root}/{path}".format(
-            root=settings.MEDIA_ROOT,
-            path=self.object.get_version_path(self.kwargs["name"]),
+        return Path(settings.MEDIA_ROOT) / self.object.get_version_path(
```

Comment:
Maybe do this once and for all in `settings/__init__.py` ?

Reply:
Why not, but I prefer to have this in a separate pull request if possible :-)